### DISCUSSION TITLE
Question: bannable?

### DISCUSSION THREAD

**Discussion by MrBoringCom** [cafali/SnapKey#23] [Q&A] [answered]
I was wondering if using this program is bannable. Am I allowed to use this in valve mm and faceit?

**Comment by Kwisss** [comment]
Had no issues with faceit, it is fine for now until they publish news its not.
Moreover this code does not jiggle like in the Wooting video, but I think they are working on it.

**Comment by cafali** [comment, marked as answer]
The code provided does not engage with any game files.
When it comes to online games, I cant really anwser this question for you.
Its the first time ever we got a feature like that in modern keyboards.
Platforms like faceit & others will need to give a statement about this feature since, it will become more popular in the future also by other keyboard brands.

**Comment by KarumGhag** [comment]
Im sure that if they allow the new razer snaptap they will have to allow this
